swap_meet/item.py

```diff
@@ -0,0 +1,38 @@
+class Item:
+    def __init__(self, category = "", condition = 0, age = None):
+        self.category = category
+        self.condition = condition
+        #...Optional Enhancement...
+        if age is None:
+            self.age = 0
+        else:
+            self.age = age
```

Comment:
This totally works! That said, in this case (unlike with an empty list) we can just set the default value to `0` rather than `None`.